ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx

```diff
@@ -0,0 +1,251 @@
+                                            icon={SearchIcon}
+                                        />
+                                    </Bullseye>
+                                </Td>
+                            </Tr>
+                        </Tbody>
+                    )}
+                    {data?.imageCVEs.length !== 0 &&
+                        data?.imageCVEs.map(
+                            (
+                                {
+                                    cve,
+                                    affectedImageCountBySeverity,
+                                    topCVSS,
+                                    affectedImageCount,
+                                    firstDiscoveredInSystem,
+                                    distroTuples,
+                                },
+                                rowIndex
+                            ) => {
+                                const isExpanded = expandedRowSet.has(cve);
+
+                                const criticalCount = affectedImageCountBySeverity.critical.total;
+                                const importantCount = affectedImageCountBySeverity.important.total;
+                                const moderateCount = affectedImageCountBySeverity.moderate.total;
+                                const lowCount = affectedImageCountBySeverity.low.total;
+                                const filteredSeverities: VulnerabilitySeverityLabel[] = [
+                                    'Critical',
+                                    'Important',
+                                    'Moderate',
+                                    'Low',
+                                ];
+                                const prioritizedDistros = sortCveDistroList(distroTuples);
+                                const scoreVersions = getScoreVersionsForTopCVSS(
+                                    topCVSS,
+                                    distroTuples
+                                );
+                                const summary =
+                                    prioritizedDistros.length > 0
+                                        ? prioritizedDistros[0].summary
+                                        : '';
+
+                                const cveURLQueryOptions = {
+                                    s: {
+                                        Image: queryObject.Image,
+                                    },
+                                    vulnerabilityState: 'DEFERRED',
+                                };
+                                // @TODO: This needs to be tested more thoroughly. Once the deferred tab shows the correct data, we should add a test for this
+                                const cveURL = getEntityPagePath(
+                                    'CVE',
+                                    cve,
+                                    omitBy(cveURLQueryOptions, (value) => value === '')
+                                );
```

Comment:
Agreed, and this will need to be the behavior throughout Workload CVEs. We might want to make `getEntityPagePath` require another parameter for the vulnerability state tab we want to link to.